tools/codegen/config.yml

```diff
@@ -274,6 +278,54 @@ resources:
           computability:
             optional: true
             computed: true
+        start_asap: # "This flag resets to false after MongoDB Cloud completes maintenance."
+          computability:
+            computed: true
+
+  # Singleton resource in project.
+  # Create uses PATCH.
+  project_settings_api:
+    read:
+      path: /api/atlas/v2/groups/{groupId}/settings
+      method: GET
+    create:
+      path: /api/atlas/v2/groups/{groupId}/settings
+      method: PATCH
+    update:
+      path: /api/atlas/v2/groups/{groupId}/settings
+      method: PATCH
+    version_header: application/vnd.atlas.2023-01-01+json
+    schema:
+      aliases:
+        # "AI" is all uppercase and does not play nice with camelCase to snake_case conversion.
+        is_data_explorer_gen_aifeatures_enabled: is_data_explorer_gen_ai_features_enabled
+        is_data_explorer_gen_aisample_document_passing_enabled: is_data_explorer_gen_ai_sample_document_passing_enabled
```

Comment:
lets sync offline, curious about this usage as currently alias config is very limited (used for path param renaming) and not considered in conversion logic.